config/locales/tr.yml

```diff
@@ -6,5 +6,8 @@ tr:
     settings:
       add_issue_matchers: İş eşleştiricileri ekleyiniz
       issues_matching_will_require_time_entry: Eşleşen iş kayıtlarında  zaman kaydı girilmesi zorunlu olacak
+      exclude_roles: Rolleri zorunlu alanı doldurma zorunluluğundan hariç tutu
```

Comment:
I guess the "tutu" needs to be "tut"

Reply:
i just used google translate, thanks for the correction